### ISSUE TITLE
Musings on alterations to Void jars.

### ISSUE THREAD

**Issue by TheLordEternal** [Azanor/thaumcraft-suggestions#357]
I understand that the void jars releasing flux when they void things are a balancing mechanic, but the thing is: If there is an byproduct, is it truly 'void'-jar?
I am aware i am arguing semantics, but the matter just doesn't sit right with me.

So my suggestion is that instead of producing Flux, the Void jar either consumes Vis from the aura, or uses Perditio/Vacuos essentia to destroy the overflow. (Supportable atm, as bowls have only Vacuos.)

Meanwhile, a counter suggestion is to make the "Overflow becomes Flux" a standard mechanic for the jars. Requiring you to be more careful with your essentia storage, as jars would always accept essentia, even when full. The extra essentia becoming overflow. When a jar becomes full it could produce much less suction than a emptier one to offset this, but would still produce some suction.

To solve the issue that would appear from this; That void jars would become superior to the regular jars, and there would be no reason to not use them all the time: the downside could be that Void jars cannot have their essentia be directly drawn from by pipes and transfusers, only filled. Preventing them from being used as the primary storage method. Manually emptying the Void jar would still produce Flux, and phials would still be able to collect the essentia from them like usual.

TL;DR:
Void jars take Vis from the aura, or essentia, instead of making Flux when they void their essentia.
Jars can overflow when full, and when they do they produce flux. (Separate but related idea)
Void jars cannot have their contents drawn from non-manually. (Separate but related idea)

**Comment by praecipitator** [comment]
Something should change allright. Currently, there is no point in using void jars, crafting stacks of regular jars and just storing them like you store excess cobblestone (or even thowing them into lava) makes much more sense.

It got even worse with the recent update, where you can use any type of essentia except vitium to condense flux. The flux condenser will not only get rid of unneeded essentia, but also REMOVES flux in the process!

I personally don't like the idea of normal jars overflowing and making flux. But I don't have any idea of how to make void jars useful again which would not involve making normal jars worse...
